src/test/javascript/spec/component/short-answer-question/short-answer-question-edit.component.spec.ts

```diff
@@ -85,7 +85,14 @@ describe('ShortAnswerQuestionEditComponent', () => {
     });
 
     it('should initialize', () => {
-        expect(component).to.be.ok;
+        component.question.text = 'This is a[-spot 12]regarding this question.\nAnother [-spot 8] is in the line above';
```

Comment:
can you add more tests? In particular some code example in which different types of braces `(`, `[`, `{` are used?
We should have at least 5-10 examples

Reply:
Now we have 7 tests which all test different use cases of using indentation (-> 4d59311)